### DISCUSSION TITLE
Whitelist CVE + image name?

### DISCUSSION THREAD

**Discussion by jonny-wg2** [aquasecurity/trivy-operator#489] [Help & Support]
Hi, Is there a way to whitelist the CVE xxxx + image name? Or is there just a way to whitelist the CVE across the board? The reason for asking is there might be a use case on why the CVE can exist on one container but should be remediated on the other.

**Comment by chen-keinan** [comment]
currently out of the box we support [ignore file](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L195) which ignore CVEs in general not for specific image , I assume we could extend this functionality and associate it to image id.

do you mind open an issue for it, probably its not such a big change
